src/workflows/server/sqlite_workflow_store.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, db_path: str) -> None:
     def _init_db(self) -> None:
         conn = sqlite3.connect(self.db_path)
         conn.execute(
-            "CREATE TABLE IF NOT EXISTS handlers (handler_id TEXT PRIMARY KEY, workflow_name TEXT, status TEXT, ctx TEXT)"
+            "CREATE TABLE IF NOT EXISTS handlers (handler_id TEXT PRIMARY KEY, workflow_name TEXT, status TEXT, ctx TEXT, handler_metadata_json TEXT)"
```

Comment:
yeah, this will need a little mini migration, otherwise existing databases will crash and not support metadata. I like this for a no-dependency sqlite version migration pattern:

https://github.com/run-llama/cloud_llama_deploy/blob/main/packages/llamactl/src/llama_deploy/cli/config/_migrations.py